airbyte-integrations/bases/connector-acceptance-test/connector_acceptance_test/tests/test_incremental.py

```diff
@@ -172,6 +177,10 @@ async def test_two_sequential_reads(
             state_input = states_1[-1].state.data
 
         # READ #2
+        if setup_container and setup_teardown_dockerfile_config.between_syncs_command:
+            detailed_logger.info(
+                await setup_container.with_exec(setup_teardown_dockerfile_config.between_syncs_command, skip_entrypoint=True).stdout()
```

Comment:
I'm wondering if we'd have to introduce a cache buster with an env var here to make sure the "between_syncs_command" is always run.

Reply:
The concern is that `with_exec` won't be run if it's already been run with the same command? The cache buster does seem like a good idea if that's the concern. That would look something like this?

```
await setup_container.with_env_variable("CACHEBUSTER", str(uuid4())).with_exec(setup_teardown_dockerfile_config.between_syncs_command, skip_entrypoint=True).stdout()
```

Reply:
Yep exactly!